### PR TITLE
DOC: Fixed PR06 error in pandas.Categorical.from_codes

### PR DESCRIPTION
**Pull request by roshni-b**
- [x] xref #28724 
- [ ] tests added / passed
- [x] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/arrays/categorical.py [hunk 1]
```diff
@@ -636,7 +636,7 @@ def from_codes(cls, codes, categories=None, ordered=None, dtype=None):
 
         Parameters
         ----------
-        codes : array-like, integers
+        codes : array-like, ints
```
[the remark this answers is not in the document]
Comment (simonjayhawkins) [reply pandas/core/arrays/categorical.py]: see https://github.com/pandas-dev/pandas/pull/28807/files#r331815427

This PR does overlap with that one.

[the remark this answers is not in the document]
Comment (roshni-b) [reply pandas/core/arrays/categorical.py]: @simonjayhawkins it does overlap. Shall I go ahead and close this PR then?

[the remark this answers is not in the document]
Comment (WillAyd) [reply pandas/core/arrays/categorical.py]: Yea I think the other one has this covered and looks active, so let's stick with that for now (@tolaa001 let us know if that is no longer being worked on). 

We have plenty of other docstrings that need fixes we would welcome your contributions on @roshni-b !

**Comment by WillAyd** [comment]
Already contained within #28807

File: pandas/core/arrays/categorical.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (roshni-b) [reply pandas/core/arrays/categorical.py]: sure sounds good! thanks @WillAyd

[the remark this answers is not in the document]
Comment (tolaa001) [reply pandas/core/arrays/categorical.py]: @WillAyd I am definitely still working on this. Will fix mine based on the comments tomorrow if that is ok. I am away from my personal laptop until tomorrow evening

**Pull request closed without merging**